examples/generate.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+from typing import List
+
+import torch
+
+from curated_transformers.generation.config import (
+    GreedyGeneratorConfig,
+    SampleGeneratorConfig,
+)
+from curated_transformers.models.attention import enable_torch_sdp
+from curated_transformers.util.auto_model import AutoGenerator
+
+parser = argparse.ArgumentParser(
+    description="Generate from a list of prompts in JSON format",
```

Comment:
It'd be good to also include an example of the format.

Reply:
But where? 🤔

Reply:
I was thinking we could add it to the description string? Doesn't it get shown in the help text?

Reply:
Added an epilogue.